### ISSUE TITLE
Alternative shoutout layout

### ISSUE THREAD

**Issue by mrclay** [kevinjardine/shoutout#27]
You can see [here](https://skitch.com/mrclay/e2c8k/shoutout-layout) the layout I gave shoutout for our COE site. Took quite a bit of CSS wrangling since I didn't want to override the view. I think this would be a better default layout.

Thoughts? Splitting the layout into several views would certainly help.

**Comment by kevinjardine** [comment]
I agree. Much better. I'm not a designer so I was hoping somebody would do something like this. If you want to rework the views, please do so and then I'll accept the full commit.

**Comment by kevinjardine** [comment]
Steve, I'm currently reworking shoutout to use the latest file uploader JS, which supports IE 10 and Android. Any chance you could submit your layout changes as a pull request so I could combine both and do a new release?

**Comment by mrclay** [comment]
Yes but stranded out of town for several days
